randovania/games/super_metroid/logic_database/Brinstar.txt

```diff
@@ -1283,13 +1320,16 @@ Extra - asset_id: 112
   > Door to Warehouse Zeela Room
       All of the following:
           Any of the following:
+              # Get through the Super Missile blocks
               Super Missile ≥ 3
               Morph Ball and Super Missile
           Any of the following:
+              # Get up to door
               Hi-Jump Boots or Space Jump
-              # https://www.youtube.com/watch?v=rHMHqTHHqHs&ab_channel=VGsWalkthrough
+              # Movement (Beginner): https://youtu.be/rHMHqTHHqHs?si=56kWaLF3RPlH5Pg9
               Movement (Beginner) and Wall Jumps (Beginner)
```

Comment:
why is this movement?

Reply:
ehhhhhhhhhhhhhhhhhhhhhhh
feels trivial to me, but its beginner so its fine i suppose